cmd/experimental/kjobctl/pkg/builder/slurm_builder.go

```diff
@@ -193,6 +195,18 @@ func (b *slurmBuilder) build(ctx context.Context) (runtime.Object, []runtime.Obj
 	job.ObjectMeta.GenerateName = ""
 	job.ObjectMeta.Name = objectName
 
+	if len(b.priority) != 0 {
```

Comment:
I also thinking about `--ignore-unknown-priority` to skip validation.

Reply:
I added the flag with name `--skip-priority-validation`. 

WDYT?

Reply:
Thanks!